packages/ui/src/working-groups/modals/CreateOpening/CreateOpening.stores.tsx

```diff
@@ -0,0 +1,51 @@
+import { Meta, Story } from '@storybook/react'
```

Comment:
There is a typo on this file's name: `CreateOpening.stores.tsx` -> `CreateOpening.stories.tsx`